### PR TITLE
Navigation typography

### PR DESCRIPTION
**Pull request by itsmebipul00**
P.S : I'll add the code snippets after this PR

### CODE REVIEW

File: default.css [hunk 1]
```diff
@@ -7,45 +7,44 @@
...
   :root {
-    /* font-sizes */
     --fs-800: 6.25rem;
     --fs-700: 3.5rem;
     --fs-600: 2rem;
     --fs-500: 1.75rem;
     --fs-400: 1.125rem;
   }
-}
+} */
```
Comment (Muzammil-khan710) [inline default.css]: remove commented code if it is of no use

File: Components/Submenus/Submenus.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.submenu-item .fa-shopping-cart,
+.fa-heart-pulse {
+  margin-left: 0.25rem;
+}
+
+.submenu-item > a {
+  padding: 0.5rem 0.68rem 0.5rem 0.25rem;
+  margin: 1rem 0.5rem 1rem 0.25rem;
```
Comment (Muzammil-khan710) [inline Components/Submenus/Submenus.css:43]: Use px for smaller values?

File: default.css (hunk 1 above)
Comment (itsmebipul00) [reply default.css]: https://discord.com/channels/914867137863118919/927786435334004787/940298016320028773

File: Components/Submenus/Submenus.css (hunk 2 above)
Comment (itsmebipul00) [reply Components/Submenus/Submenus.css:43]: https://discord.com/channels/914867137863118919/927786435334004787/940298016320028773

File: default.css (hunk 1 above)
Comment (itsmebipul00) [reply default.css]: done

**Review by Muzammil-khan710 — APPROVED** [review]

**Review by zoef-yt — APPROVED** [review]
Looks good to me. Cleancode. Good to merge.

File: default.css [hunk 3]
```diff
@@ -146,9 +130,15 @@ select {
 }
 
 .f-col {
+  display: flex;
   flex-direction: column;
 }
 
+.f-row {
+  display: flex;
+  flex-direction: row;
```
Comment (iRohitGaur) [inline default.css:139]: by default the direction is row. unless you want to override a column direction, you can avoid it

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: default.css (hunk 3 above)
Comment (itsmebipul00) [reply default.css:139]: Thankyou for pointing this out!

Comment (itsmebipul00) [reply default.css:139]: noted

**Pull request merged**